internal/services/redis/redis_linked_server_resource_test.go

```diff
@@ -61,7 +61,11 @@ func (t RedisLinkedServerResource) Exists(ctx context.Context, clients *clients.
 func (RedisLinkedServerResource) basic(data acceptance.TestData) string {
 	return fmt.Sprintf(`
 provider "azurerm" {
-  features {}
+  features {
+    resource_group {
+      prevent_deletion_if_contains_resources = false
```

Comment:
this would indicate something isn't being deleted correctly which would cause issues for real users - rather than adding this workaround, can we fix the issue here?